src/main/java/com/solidstategroup/bullettrain/BulletTrainClient.java

```diff
@@ -226,6 +226,60 @@ public Trait updateTrait(FeatureUser user, Trait toUpdate) {
         return postUserTraits(user, toUpdate);
     }
 
+    /**
+     * <p>
+     * Create or update a list of user Traits for given user identity.
+     * </p>
+     * <p>
+     * Please note this will override any existing identity with given list.
+     * </p>
+     *
+     * @param user   a user in context
+     * @param traits a list of Trait object to be created or updated
+     * @return a list of added Trait objects
+     */
+    public List<Trait> addTraitsForIdentity(FeatureUser user, List<Trait> traits) {
```

Comment:
I think this should be more like `identifyUserWithTraits`, what have we called it in the JS library @kyle-ssg ?

Reply:
For the JS client I'll just be making traits an optional param to identify.

Reply:
Yeah, that makes to me. Can we do that here @p-maks ?

Reply:
changed